compiler/record-minmax/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+nnas_find_package(Boost REQUIRED COMPONENTS program_options)
```

Comment:
Please remove ``REQUIRED`` and use ``QUIET`` instead

Reply:
(Optional) I personally recommend not to use boost program option due to some porting issue, but that is just a minor issue.

Reply:
> Please remove REQUIRED and use QUIET instead

I've changed it. Thanks.